From f4100a5e9d55bc778445cbc0b612c8cbe4215954 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 22 Feb 2021 11:16:11 -0500
Subject: [PATCH] Fix two errant 'shim,0' outdated sbat cases.

Two places we missed still have 0 for an sbat version - one doc and one
in our data csv.

This fixes those.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 SBAT.md       | 2 +-
 data/sbat.csv | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/SBAT.md b/SBAT.md
index ce4ef69..0d54678 100644
--- a/SBAT.md
+++ b/SBAT.md
@@ -323,7 +323,7 @@ At the same time, we're all shipping the same `shim-16` codebase, and in our
 `shim` builds, we all have the following in `.sbat`:
 ```
 sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
-shim,0,UEFI shim,shim,16,https://github.com/rhboot/shim
+shim,1,UEFI shim,shim,16,https://github.com/rhboot/shim
 ```
 
 How to add .sbat sections
diff --git a/data/sbat.csv b/data/sbat.csv
index 08a2459..ad838f2 100755
--- a/data/sbat.csv
+++ b/data/sbat.csv
@@ -1,2 +1,2 @@
 sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
-shim,0,UEFI shim,shim,0,https://github.com/rhboot/shim
+shim,1,UEFI shim,shim,1,https://github.com/rhboot/shim
-- 
2.30.0

